README.md

```diff
@@ -25,38 +25,39 @@ Then you have to choose between the different configs, depending on the project
   - react
   - react-native
 
-You'll also need to include eslint & any other plugins/parsers that are needed in your package.json.
-Particularly:
+You'll also need to include eslint & any other plugins/parsers that are needed in your package.json. Particularly:
```

Comment:
If they are dependencies of our config, there's no need to add them as dependencies of your project (in node 5+).